deftree/build_deftree.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/TuneLab/go-truss/deftree/svcparse"
-	"github.com/TuneLab/go-truss/truss/protostage"
+	"github.com/TuneLab/go-truss/truss/protostuff"
```

Comment:
Well I am importing a packaged named `proto` in `protostuff` but if you think it would be fine to just call it `proto` I would be down for that.